endpoints/authentication/src/main.cpp

```diff
@@ -349,7 +349,8 @@ namespace irods::http::handler
 					// Issue token?
 					static const auto seconds =
 						irods::http::globals::configuration()
-							.at(nlohmann::json::json_pointer{"/http_server/authentication/basic/timeout_in_seconds"})
+							.at(nlohmann::json::json_pointer{
+								"/http_server/authentication/openid_connect/timeout_in_seconds"})
```

Comment:
Was this supposed to change from basic to openid_connect?

Reply:
Yes. These are for the OIDC code paths. We added a the timeout option to separate Basic config from OIDC config.

Reply:
Okay, just making sure.